packages/providers/src/providers/localai.ts

```diff
@@ -0,0 +1,24 @@
+import type { ChatProvider, EmbedProvider, ModelProvider, ProviderOptions, ProviderResult } from '../types'
+
+import { generateCRO } from '../utils/generate-cro'
+
+export const createLocalAI = (userOptions?: ProviderOptions<false>):
+  /** @see {@link https://localai.io/model-compatibility/} */
+  ChatProvider<({} & string)>
+  & EmbedProvider<({} & string)>
```

Comment:
```suggestion
  ChatProvider
  & EmbedProvider
```

If auto-completion is not provided, generics is not required.